### PR TITLE
DIO-67 - Add Windows WSL documentation

### PR DESCRIPTION
**Pull request by michaelloosen**
To support Windows users in setting up instant for Windows development

DIO-67

### CODE REVIEW

File: docs/docs/how-to/setup-intant-on-windows-wsl.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+```
+```sh
+sudo apt-get install git
+```
+
+## 4. Docker Desktop Setup
+
+Download and install Docker Desktop from:
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
```
Comment (MattyJ007) [inline docs/docs/how-to/setup-intant-on-windows-wsl.mdx:44]: Help out Linux users.
I tried user the terminal cli - it's a schlep so even though I don't use the gui the setup is great

```suggestion
> Docker Desktop install `docker-compose` by default and sets up docker without sudo in the Linux terminal

```

File: docs/docs/how-to/setup-intant-on-windows-wsl.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+> Tip: Do not clone the repository or store any important files in the `/mnt/wsl` directory as this directory gets cleared every time the Windows `LxssManager` service gets restarted (every time you restart Windows).
+
+Build the Docker images using:
+
+```sh
+yarn docker:instant build
+```
+
```
Comment (MattyJ007) [inline docs/docs/how-to/setup-intant-on-windows-wsl.mdx:86]: Could you please add in the start up command for core. And perhaps an OpenHIM Console check

```
yarn docker:instant init core
```

File: docs/docs/how-to/setup-intant-on-windows-wsl.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  - How to
+  - Windows
+  - WSL
+description: How to setup Instant OpenHIE for development on Windows with WSL
+---
+
+:::info
+The Instant OpenHIE architecture, codebase, and documentation are under active development and are subject to change. While we encourage adoption and extension of the Instant OpenHIE framework, we do not consider this ready for production use at this stage.
+:::
+
```
[the remark this answers is not in the document]
Comment (MattyJ007) [reply docs/docs/how-to/setup-intant-on-windows-wsl.mdx:16]: And perhaps also a prerequisites section just listing the Windows 10 version needed, fast internet, and the 16GB RAM

**Review by MattyJ007 — CHANGES_REQUESTED** [review]
@michaelloosen Looking good - starts up well. 
I've left a few comments about where there could be a bit more detail. When I was going through the instructions I had to make a few assumptions. I think specifying the tested path exactly would be helpful. 
Also, did you limit the amount of memory that WSL could use?

File: docs/docs/how-to/setup-intant-on-windows-wsl.mdx (hunk 2 above)
Comment (michaelloosen) [reply docs/docs/how-to/setup-intant-on-windows-wsl.mdx:86]: @MattyJ007  I've added the start up command, could you please clarify what you mean with "an OpenHIM Console check"?

Comment (MattyJ007) [reply docs/docs/how-to/setup-intant-on-windows-wsl.mdx:124]: Lols, sorry  - I mean just go to http://localhost:9000 and login to check that the OpenHIM has spun up properly.

**Review by MattyJ007 — APPROVED** [review]
Great work! 🥇

**Pull request merged**